joins/index-join_base_idx.json

```diff
@@ -0,0 +1,43 @@
+{% set p_include_non_serverless_index_settings = (include_non_serverless_index_settings | default(build_flavor != "serverless")) %}
+
+{
+  "settings": {
+    {# non-serverless-index-settings-marker-start #}{%- if build_flavor != "serverless" or serverless_operator == true -%}
+    "index.requests.cache.enable": false
+    {%- endif -%}{# non-serverless-index-settings-marker-end #}
+  },
```

Comment:
Let's default to 0 replicas with:
```
"index.number_of_replicas": {{number_of_replicas | default(0)}},
```

Like in: https://github.com/elastic/rally-tracks/blob/d5697bd8d44c4dd099851993d59505c342e847be/geopoint/index.json#L5-L11

Unless the track is _not_ supposed to be used in serverless in case of which this is unnecessary boilerplate.

Please update remaining index files accordingly.

Reply:
Doing it for the base index, but I cannot do it for the "lookup" indexes. They will have N replicas as the number of nodes, that's specific to the type of index. I hope it's not a problem for the CI